### PR TITLE
finish task 6

### PR DESCRIPTION
**Pull request by XingWei0920**
I just finished task 6.

### CODE REVIEW

File: components/ArticleProfile.jsx [hunk 1]
```diff
@@ -29,15 +39,24 @@ const ArticleProfile=()=>{
...
+        <p>Topic: {article.topic}</p>
+        <p>Votes: {article.votes}</p>
+        <p>Title: {article.title}</p>
+        <p> Body: {article.body}</p>
+        <p> Created At: {new Date(article.created_at).toUTCString()}</p>
+        <p> Comment Count: {article.comment_count}</p>
         <img src={article.article_img_url} alt="image" width="100" height="100"></img>
         </h2>  
+
+        <h2 className="itemBlock">All Comments Lists</h2>
```
Comment (ghost) [inline components/ArticleProfile.jsx]: Is this title necessary? It is the same font size as your entire article and has no visual hierarchy at all. On an article with no comments (number 37) it is quite misleading to have a title saying there would b comments when there aren't any.

Consider changing this for something more meaningful. You could look at conditional logic to change this text depending whether there are comments or not.

File: components/CommentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import './ItemCard.css'
+
+
+const CommentCard=(props)=>{
+    const {article}=props
+    console.log(props)
```
Comment (ghost) [inline components/CommentCard.jsx]: Don't leave console logs in code that you want reviewing.

File: components/ArticleProfile.jsx [hunk 3]
```diff
@@ -20,6 +21,15 @@ const ArticleProfile=()=>{
         })
       },[])
 
+    useEffect(()=>{
+      getCommentsOfAnArticle(article_id)
+        .then((response)=>{ 
+          console.log(response)
```
Comment (ghost) [inline components/ArticleProfile.jsx]: Dont leave console logs in code that is being reviewed.

**Review by ghost — APPROVED** [review]
I can see all the comments and no errors on an article with no comments. 

Several points for you to consider to improve this ticket. I really urge you to read this article https://www.refactoringui.com/previews/labels-are-a-last-resort You are labelling **every** bit of text and that is not necessary. A comment section should not need a label of body to tell me that it is the comment text. The visual hierarchy of your layout and text sizes should tell me that.

**Review by nc-modou — COMMENTED** [review]
Good response to feedback, feel free to merge.

You could do a message for articles with no comments, but because the site still looks fine and it says comment count:0, I'd consider this feedback enough in that case

**Pull request merged**